packages/web/src/layouts/pool/pool-remove/containers/remove-liquidity-container/RemoveLiquidityContainer.tsx

```diff
@@ -27,15 +29,44 @@ const RemoveLiquidityContainer: React.FC = () => {
     },
   });
 
+  const positionList: PoolPositionModel[] = useMemo(() => {
+    if (!allPosition) return [];
+
+    return allPosition.map((position: PoolPositionModel) => {
+      return {
+        ...position,
+        tokenABalance: String(makeDisplayTokenAmount(position.pool.tokenA, position.tokenABalance || 0)),
+        tokenBBalance: String(makeDisplayTokenAmount(position.pool.tokenB, position.tokenBBalance || 0)),
+
+        claimedRewards: position.claimedRewards.map(reward => {
+          return {
+            ...reward,
+            claimedAmount: String(makeDisplayTokenAmount(reward.rewardToken, reward.claimedAmount || 0) ?? 0),
+          };
+        }),
+        rewards: position.rewards.map(reward => {
+          const rewardToken = reward.rewardToken;
+
+          return {
+            ...reward,
+            accuReward1D: String(makeDisplayTokenAmount(rewardToken, reward.accuReward1D || 0) ?? 0),
+            claimableAmount: String(makeDisplayTokenAmount(rewardToken, reward.claimableAmount || 0) ?? 0),
+            totalAmount: String(makeDisplayTokenAmount(rewardToken, reward.totalAmount || 0) ?? 0),
+          };
+        }),
+      };
+    });
+  }, [allPosition]);
```

Comment:
This part is being used in multiple places, such as the `positionList` in `StakePositionContainer.tsx` file.

How about extracting it into a function like `transformPosition`?